doc/UserGuide.md

```diff
@@ -42,6 +42,14 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Put a `p` before the phone / email / address prefixes to mark it as `private`. `private` details can only
 > be seen using the `viewall` command.
 > 
+> Address must in the format `a/BLOCK, STREET, UNIT, POSTAL_CODE`
```

Comment:
Well done on updating user guide! & commit it as a separate commit! Hope you don't mind me showing this commit to the next class :)